fedimint-server/src/net/api.rs

```diff
@@ -103,7 +103,8 @@ fn attach_endpoints(
             .register_async_method(path, move |params, state| async move {
                 let params = params.one::<serde_json::Value>()?;
                 let fedimint = &state.fedimint;
-                let dbtx = fedimint.database_transaction().await;
+                let mut dbtx = fedimint.database_transaction().await;
+                dbtx.surpress_warning();
```

Comment:
I'm not super crazy about this solution (surpressing the warning message). Ideally it should be possible to pass ReadOnlyDatabaseTransactions into api_endpoints so that it is not necessary to surpress the warning message. I couldn't figure out how to change HandlerFn though to accept a generic R or RW database transaction. I didn't really want to add another trait just to support this, not sure if there's another way though.

Reply:
Removed in new iteration